test/TestWitnetDecoderLib.sol

```diff
@@ -121,6 +121,18 @@ contract TestWitnetDecoderLib {
     );
   }
 
+  function testBytes32DecodeValueFrom31bytes() external {
+    // TODO
+  }
+
+  function testBytes32DecodeValueFrom32bytes() external {
+    // TODO
+  }
+
+  function testBytes32DecodeValueFrom33bytes() external {
+    // TODO
+  }
```

Comment:
???

Reply:
My bad 🙈

Reply:
This is my only objection!

Reply:
Now fixed in https://github.com/witnet/witnet-solidity-bridge/pull/209/commits/b2c8881c371f855a0943a361f1dafeabe1253497.